tensorflow/lite/kernels/internal/reference/space_to_depth.h

```diff
@@ -43,7 +43,7 @@ inline void SpaceToDepth(const tflite::SpaceToDepthParams& op_params,
   const int output_height = output_shape.Dims(1);
   const int output_batch = output_shape.Dims(0);
 
-  const int32 block_size = op_params.block_size;
+  const int32_t block_size = op_params.block_size;
```

Comment:
@rkuester @petewarden 

note that changes to shared TFL/TFLM code must be made in upstream TF, not in the tflite-micro repository.

We will soon be adding a check for that.

Any changes in tflite-micro will break the automated sync between the two repos.

Reply:
Example of a broken sync: https://github.com/tensorflow/tflite-micro/pull/209